### DISCUSSION TITLE
Ip adres internal and not external

### DISCUSSION THREAD

**Discussion by huizebruin** [inventree/InvenTree#8003] [General]
Hello. 

How can i set that inventree go to a internal ip adres,  and not to the external ip adres of my home. 

I did install by the script on a debian 12 machine. 

I love your work of this software. 
And will look how i can help to translate it into Dutch more.

**Comment by SchrodingersGat** [comment]
You should just be able to set the `INVENTREE_SITE_URL` to your internal IP, e.g. `http://192.168.1.123`

https://docs.inventree.org/en/stable/start/config/#server-access

**Reply by huizebruin** [reply]
in my config.yaml file ?

**Reply by SchrodingersGat** [reply]
Yes, but it really depends on how you have setup your installation. If you have an environment variable configured it will override the value in the configuration file.
